src/Storages/MergeTree/MergeTreeBackgroundExecutor.h

```diff
@@ -185,6 +267,18 @@ class MergeTreeBackgroundExecutor final : boost::noncopyable
             pool.scheduleOrThrowOnError([this] { threadFunction(); });
     }
 
+    MergeTreeBackgroundExecutor(
+        String name_,
+        size_t threads_count_,
+        size_t max_tasks_count_,
+        CurrentMetrics::Metric metric_,
+        std::string_view policy)
+        requires requires(Queue queue) { queue.updatePolicy(policy); } // Because we use explicit template instantiation
```

Comment:
`RoundRobinRuntimeQueue` doesn't have `updatePolicy`, we have a compile failure here